apps/server/src/modules/learnroom/service/common-cartridge-export.service.spec.ts

```diff
@@ -183,10 +185,17 @@ describe('CommonCartridgeExportService', () => {
 			});
 
 			it('should add column', async () => {
-				const { archive, columnBoard } = await setup();
+				const { archive, column } = await setup();
 				const manifest = getFileContent(archive, 'imsmanifest.xml');
 
-				expect(manifest).toContain(createXmlString('title', (columnBoard.children[0] as Column).title));
+				expect(manifest).toContain(createXmlString('title', column.title));
```

Comment:
I think it's easier to read this way to directly use the objects. Otherwise we would need to write ```((columnBoard.children[0] as Column).children[0] as Card).title``` for cards